doc/src/fpga_tools/ofs.uio/ofs.uio.md

```diff
@@ -0,0 +1,176 @@
+                  [--mailbox-read offset] [--mailbox-dump address size] [--mailbox-write address value]`<br>
+
+`ofs.uio [--uio uiox] [--peek offset]`<br>
+`ofs.uio [--uio uiox] [--poke offset value] `<br>
+`ofs.uio [--uio uiox] [--mailbox-read address] `<br>
+`ofs.uio [--uio uiox] [--mailbox-write address value] `<br>
+`ofs.uio [--uio uiox] [--mailbox-dump address size] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--peek offset]`<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--poke offset value] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--mailbox-read address] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--mailbox-write address value] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--mailbox-dump address size] `<br>
+
+
+## DESCRIPTION ##
+
+```ofs.uio``` is a tool that provides user space access to DFL UIO devices,
+command line options like peek, poke, mailbox-read, mailbox-write, mailbox-dump to 
+access Configuration and Status Registers (CSRs).
+
+
+##  OPTIONS ##
+
+#### Peek ####
+Peek/Read UIO CSR offset<br>
+`ofs.uio [--uio uio] [--peek offset]`<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--peek offset]`<br>
+
+#### Poke ####
+Poke/Write value to UIO CSR offset<br>
+`ofs.uio [--uio uio] [--poke offset value] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--poke offset value] `<br>
+
+#### Mailbox Read ####
+Reads CSR address offset using CSR using mailbox<br>
+`ofs.uio [--uio uio] [--mailbox-read address] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--mailbox-read address] `<br>
+
+#### Mailbox Write ####
+Writes value to CSR address offset using CSR using mailbox <br>
+`ofs.uio [--uio uio] [--mailbox-write address value] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--mailbox-write address value] `<br>
+
+#### Mailbox Dump ####
+Reads/Dumps block size of CSR address offset using CSR using mailbox<br>
+`ofs.uio [--uio uio] [--mailbox-dump address size] `<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--mailbox-dump address size] `<br>
+
+#### Bit size ####
+Read/Write bit-field 8,16,32,64 sizes<br>
+`ofs.uio [--uio uio] --bit-size 8 [--peek offset]`<br>
+`ofs.uio [--uio uio] --bit-size 32 [--peek offset]`<br>
+
+#### PCIe Address ####
+PCIE_ADDR PCIe address of FPGA device<br>
+`ofs.uio [--pcie-address PCIE_ADDRESS] [--feature-id FEATURE_ID] [--peek offset]`<br>
+
+#### UIO region index ####
+UIO region index, default region index is 0 <br> 
+`ofs.uio [--uio uio] --region-index 0 [--peek offset]`<br>
+`ofs.uio [--uio uio] --region-index 1 [--peek offset]`<br>
+
+#### Mailbox command status csr offset ####
+mailbox command status csr address offset, default value PCIe SS mailbox offset 0x28 <br> 
```

Comment:
Not familiar with SS in this context.  What can it expand to?

Reply:
fixed